src/main/groovy/com/entagen/jenkins/JenkinsApi.groovy

```diff
@@ -224,7 +225,8 @@ class JenkinsApi {
         http.handler.failure = { resp ->
             def msg = "Unexpected failure on $jenkinsServerUrl$path: ${resp.statusLine} ${resp.status}"
             status = resp.statusLine.statusCode
-            throw new Exception(msg)
+            println msg
+            this.success = false
```

Comment:
For example if you look here
https://github.com/revdotcom/jenkins-build-per-branch/blob/master/src/main/groovy/com/entagen/jenkins/JenkinsJobManager.groovy#L93

You will see that when deleting a job it first attempts to delete the workspace and then deletes the job. I would think this is where we want to make the change. (Note: I also removed my auto retry code.)
```
    public void deleteDeprecatedJobs(List<String> deprecatedJobNames) {
        if (!deprecatedJobNames) return
        println "Deleting deprecated jobs:\n\t${deprecatedJobNames.join('\n\t')}"
        deprecatedJobNames.each { String jobName ->
            try {
                jenkinsApi.wipeOutWorkspace(jobName)
                jenkinsApi.deleteJob(jobName)
            }
            catch(Exception ex) {
                //set global failure flag
               failed = true
            }
        }
    }
```